xdsl/interactive/app.py

```diff
@@ -167,24 +222,31 @@ def watch_current_module(self):
 
         self.output_text_area.load_text(output_text)
 
-    def on_mount(self) -> None:
-        """Configure widgets in this application before it is first shown."""
-
-        # register's the theme for the Input/Output TextArea's
-        self.input_text_area.register_theme(InputApp._DEFAULT_THEME)
-        self.output_text_area.register_theme(InputApp._DEFAULT_THEME)
-        self.input_text_area.theme = "my_theme_design"
-        self.output_text_area.theme = "my_theme_design"
-
-        self.query_one("#input_container").border_title = "Input xDSL IR"
-        self.query_one("#output_container").border_title = "Output xDSL IR"
-        self.query_one(
-            "#passes_list_view"
-        ).border_title = "Choose a pass or multiple passes to be applied."
-        self.query_one("#selected_passes").border_title = "Selected passes/query"
-
-        for n, _ in ALL_PASSES:
-            self.passes_list_view.append(ListItem(Label(n), name=n))
+    def trigger(self):
```

Comment:
My high-level model of how this works is "when either the current module or condense mode changes, update the current available passes". My understanding of textual is that the idiomatic way to do this is with a `compute_current_condensed_pass_list`, which would call the helper function above to update the list. Could you please give that a go?